src/app/features/auth/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,86 @@
+<form
+        class="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 w-400px"
```

Comment:
Setting width of a component inside the component itself is not a good idea. I might want it to be 400px in one place, but 600px – in another. What would I do then?